src/inventory_ui.h

```diff
@@ -579,7 +574,7 @@ class inventory_selector
         /** @return an entry from all entries by its invlet */
         inventory_entry *find_entry_by_invlet( int invlet ) const;
 
-        inventory_entry *find_entry_by_coordinate( point coordinate ) const;
+        inventory_entry *find_entry_by_coordinate( const point coordinate ) const;
```

Comment:
```suggestion
        inventory_entry *find_entry_by_coordinate( point coordinate ) const;
```
Top level const in parameter types of a function deceleration is ignored anyway. It has no meaning to the caller.